src/main/resources/templates/signup.html

```diff
@@ -58,5 +66,6 @@ <h2 class="text-center mb-4">Sign up</h2>
 
     <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/js/bootstrap.bundle.min.js" 
       integrity="sha384-geWF76RCwLtnZ8qwWowPQNguL3RmwHVBC9FhGdlKrxdiJJigb/j/68SIy3Te4Bkz" crossorigin="anonymous"></script>
+      <script defer src="../static/js/signup-account.js"></script>
```

Comment:
Since it's a Thymeleaf template, the script doesn't get added this way. Do that instead:

```suggestion
    <script th:src="@{/js/signup.js}" src="/static/js/signup.js"></script>
```